modules/AlphaWalletFoundation/AlphaWalletFoundation/CoinTicker/CoinGecko/CoinGeckoTickersFetcher.swift

```diff
@@ -183,3 +183,28 @@ extension CoinGeckoTickersFetcher.HistoryToPeriod: Reorderable {
     typealias OrderElement = ChartHistoryPeriod
     var orderElement: ChartHistoryPeriod { return period }
 }
+
+extension CoinGeckoTickersFetcher {
```

Comment:
Do you want to move it into the CoinGeckoTickersFetcher type definition instead of creating a new extension? Seems easier to find it. I'd use extension if it's good to group properties/functions together for a single purpose (kind of like pragma or marks)